src/libraries/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509Chain.cs

```diff
@@ -62,12 +62,12 @@ public X509ChainPolicy ChainPolicy
             }
         }
 
-        public X509ChainStatus[] ChainStatus
+        public X509ChainStatus[]? ChainStatus
```

Comment:
I don't think this can ever actually be null... I don't think that _pal is ever in a state where the Linux instance would not yet have assigned the ChainStatus field; the Windows PAL never can return null; and I didn't check macOS.

Reply:
It looks like macOS will initialize in the `BuildChain` method as well. I'll change it.